v2/pkg/engine/datasource/grpc_datasource/grpc_datasource_test.go

```diff
@@ -1803,6 +1858,47 @@ func Test_DataSource_Load_WithNullableFieldsType(t *testing.T) {
 				}
 			},
 		},
+		{
+			name:  "Query nullable fields type with all fields and aliases",
```

Comment:
With all aliased fields? current name feels like we should have all fields and than all aliases